pkg/innerring/processors/governance/events.go

```diff
@@ -9,3 +17,25 @@ func (s Sync) MorphEvent() {}
 func NewSyncEvent() Sync {
```

Comment:
process_epoch.go still generates internal event to sync validators on each epoch. Maybe it is redundant because of RoleManagement contract notifications, what do you think? 

```go
// pkg/innerring/processors/netmap/process_epoch.go
func (np *Processor) processNewEpoch(epoch uint64) {
	...
	np.handleAlphabetSync(governance.NewSyncEvent())
}
```

Reply:
It is not redundant, because committee changes are limited by `1/3n-1` of new keys. Thus it may require several synchronization rounds, however RoleManagement contract produces notification only once.